src/main/java/com/socialmedia/service/PostService.java

```diff
@@ -72,4 +73,28 @@ public Post delete(Long id) {
         .sorted((p1, p2) -> (int) (p1.getDate() - p2.getDate()))
         .collect(Collectors.toList());
   }
+
+  public void updateLikes(Long postId, ApplicationUser author) {
+    AtomicBoolean isPresent = new AtomicBoolean(false);
```

Comment:
Why atomic boolean?

Reply:
Variable used in lambda expression should be final or effectively final (Convert to atomic or transform "isPresent" into final one element array)

Reply:
made without AtomicBoolean